resources/functionMap.php

```diff
@@ -261,7 +261,7 @@
 'array_change_key_case' => ['array', 'input'=>'array', 'case='=>'int'],
 'array_chunk' => ['list<array>', 'input'=>'array', 'size'=>'positive-int', 'preserve_keys='=>'bool'],
 'array_column' => ['array', 'array'=>'array', 'column_key'=>'mixed', 'index_key='=>'mixed'],
-'array_combine' => ['array|false', 'keys'=>'array', 'values'=>'array'],
+'array_combine' => ['array|false', 'keys'=>'array<array-key>', 'values'=>'array'],
```

Comment:
while here I used the same fix here - I think `array-key` better conveys the idea of the type beeing required